src/test/resources/org/broadinstitute/barclay/help/templates/BashTestDoclet/bash-completion.template.ftl

```diff
@@ -0,0 +1,343 @@
+
+####################
+# Tab completion file to allow for easy use of this tool with the command-line (bash/zsh)
+####################
+
+<#include "bash-completion.macros.ftl"/>
+
+####################################################################################################
+
+# High-level caller/dispatch script information:
```

Comment:
Yeah - I used this as a test bed for the GATK launch script, but I can make it a little more generic.  In practice, all that's necessary is to change this section at the top of the template (lines 12-20) to be application-specific.  

Right now it assumes there's a central dispatch / launcher script that is used to run the other tools.